koku/masu/database/sql/aws/remove_stale_enabled_tags.sql

```diff
@@ -1,5 +1,12 @@
 -- Delete stale enabled keys
+WITH cte_tag_mapping AS(
+    SELECT distinct
```

Comment:
question: What is the `distinct` here doing, shouldn't the `array_agg` just turn all of the child and parent uuids into one record anyway?

Reply:
Awesome feedback @cgoodfred! Thank you! Let me push up a change.